client.go

```diff
@@ -335,15 +335,34 @@ func (c *ovndb) reconnect() {
 	}()
 }
 
-func (c *ovndb) MonitorTables(jsonContext interface{}) (*libovsdb.TableUpdates, error) {
-	// get the table list based on the DB
+// checkSchemaTables checks whether tables in
+// NBTablesOrder / SBTablesOrder exists in current ovn-db schema
+func (c *ovndb) checkSchemaTables() []string {
+	schemaTables := make([]string, 0)
 	var tables []string
+
+	// get the table list based on the DB
 	if c.db == DBNB {
 		tables = NBTablesOrder
 	} else {
 		tables = SBTablesOrder
 	}
 
+	dbSchema, err := c.client.GetSchema(c.db)
```

Comment:
why are you calling c.client.GetSchema(), again? 
c.client.Schema[c.db] should already have that, isn't it?

Reply:
you can then remove lines 352-355

Reply:
Missed that, use that.